src/driver/postgres/PostgresQueryRunner.ts

```diff
@@ -1739,10 +1739,12 @@ export class PostgresQueryRunner extends BaseQueryRunner implements QueryRunner
     }
 
     protected createViewSql(view: View): Query {
+        const materializedClause = view.materialized ? "MATERIALIZED " : "";
+
         if (typeof view.expression === "string") {
-            return new Query(`CREATE VIEW ${this.escapePath(view)} AS ${view.expression}`);
+            return new Query(`CREATE ${materializedClause}VIEW "${view.name}" AS ${view.expression}`);
```

Comment:
what is the point of removing `this.escapePath` here?

Reply:
Oops, didn't notice that! Will amend, good catch :smile: 

(I'll be honest, it was copy pasta from the previous PR :wink:)

Reply:
Looks like that change causes a syntax error. I'll see if I can get this sorted over the next day or so.